string/string.c

```diff
@@ -460,3 +460,38 @@ char *strncat(char *dest, const char *src, size_t n)
 
 	return dest;
 }
+
+
+size_t strlcpy(char *dst, const char *src, size_t size)
+{
+	size_t len = 0;
+
+	if (size) {
+		while (--size && *src) {
+			*(dst++) = *(src++);
+			len++;
+		}
+		*dst = '\0';
+	}
+
+	while (*(src++))
+		len++;
+	return len;
+}
+
+
+size_t strlcat(char *dst, const char *src, size_t size)
+{
+	size_t srclen = strlen(src);
+	size_t dstlen = strnlen(dst, size);
+
+	if (dstlen == size)
```

Comment:
We should perform operation anyway, but resulting string should be not NULL terminated